frontend/src/components/viewers/MetricsVisualizations.tsx

```diff
@@ -98,6 +115,29 @@ function getMetricsAvailableArtifacts(artifacts: Artifact[]): Artifact[] {
     .map(x => x.artifact);
 }
 
+function getVerifiedMetricsArtifacts(
+  artifacts: Artifact[],
+  artifactTypes: ArtifactType[],
+): Artifact[] {
+  if (!artifacts || !artifactTypes) {
+    return [];
+  }
+  // Reference: https://github.com/kubeflow/pipelines/blob/master/sdk/python/kfp/dsl/io_types.py#L104
+  // system.Metrics contains scalar metrics.
+  let metricsArtifacts = filterArtifactsByType('system.Metrics', artifactTypes, artifacts);
```

Comment:
nit: use `const` when the var does not need a change

Reply:
Make sense! Updated.